qiskit_metal/renderers/renderer_gmsh/gmsh_renderer.py

```diff
@@ -0,0 +1,894 @@
+from typing import Union, List
+from collections import defaultdict
+import pandas as pd
+import gmsh
+import numpy as np
+
+from ..renderer_base import QRendererAnalysis
+from ...designs.design_base import QDesign
+from .gmsh_utils import Vec3D, Vec3DArray, line_width_offset_pts, render_path_curves
+from ...draw.basic import is_rectangle
+from ..utils import get_min_bounding_box  # TODO: remove and put the new one written by @PritiShah
+from ...toolbox_metal.parsing import parse_value
+from ...toolbox_python.utility_functions import clean_name, bad_fillet_idxs
+from ... import Dict
+
+
+class QGmshRenderer(QRendererAnalysis):
```

Comment:
QRenderer as parent instead? If not needing all of the abstractiong from QRendererAnalysis.

Reply:
When Gmsh is subclasses, then I believe the abstraction would be used from QRendererAnslysis.

Reply:
![image](https://user-images.githubusercontent.com/74020801/180868394-0752b88f-d544-40c5-9d6b-ef1b9f7a4d6c.png)
This will be needed from Renderer Base.